README.md

```diff
@@ -92,14 +92,20 @@ scrape_configs:
         target_label: instance
       - target_label: __address__
         replacement: <<REDIS-EXPORTER-HOSTNAME>>:9121
+  ## it's also necessary
+  ## config for scraping the exporter itself
+  - job_name: 'redis_exporter'
```

Comment:
This needs another name, we already have `redis_exporter` in line 83.
How about changing line 83 to `redis_instances`?

Reply:
yes, you are right, it's my  careless